src/modules/Dashboard/components/TakeNowModal/TakeNowModal.tsx

```diff
@@ -222,7 +284,16 @@ export const useTakeNowModal = ({ dataTestId }: UseTakeNowModalProps) => {
           url.searchParams.set('respondentId', loggedInUser.userId);
         }
 
-        setActivity(null);
+        track('Multi-informant Start Activity click', {
+          [MixProperties.SourceAccountType]: getAccountType(sourceSubject),
+          [MixProperties.TargetAccountType]: getAccountType(targetSubject),
+          [MixProperties.InputAccountType]: getAccountType(
+            isSelfReporting ? sourceSubject : loggedInUser,
+          ),
+          [MixProperties.IsSelfReporting]: isSelfReporting,
```

Comment:
The `isSelfReporting` variable only represents the state of the checkbox, but technically we're still self-reporting if the source subject and the logged in user are the same
```suggestion
          [MixProperties.IsSelfReporting]:
            isSelfReporting || (loggedInUser && sourceSubject.id === loggedInUser.id),
```

Reply:
Good catch!